src/main/java/com/github/pereiratostain/Main.java

```diff
@@ -58,6 +58,6 @@ public static List<Entity> asm() throws IOException {
         return visitors.stream().map(Visitor::getEntity).toList();
       }
     }
-    return null;   //FIXME there is something wrong here !
+    return new ArrayList<>();
```

Comment:
use List.of() instead, this is a constant so no allocation (in fact, one shared by all calls to List.of() with no argument)